buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -242,6 +243,28 @@ public void beforeExecute(Task task) {}
         );
     }
 
+    private void stopCluster(ElasticsearchCluster each, boolean taskFailed) {
+        if (allowClusterToSurvive) {
+            logger.info("Not stopping clusters, disabled by property");
+            if (taskFailed) {
+                // task failed or this is the last one to stop
+                for (int i=1 ; ; i += i) {
+                    logger.lifecycle(
+                        "No more test clusters left to run, going to sleep because {} was set," +
+                            " interrupt to stop clusters.", TESTCLUSTERS_INSPECT_FAILURE
```

Comment:
We should be more specific. What do we mean by "interrupt"? I assume this means `CTRL+C`?

Reply:
The goal is to have the cleanup run after, so one doesn't have to hunt for them and stop them.